parts/chapters/subsections/12.3/WDFAC.fodt

```diff
@@ -4481,7 +4481,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
       <table:covered-table-cell/>
      </table:table-row>
     </table:table>
-    <text:p text:style-name="P58">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">12.3.259.1</text:sequence>: <text:span text:style-name="T22"><text:a xlink:href="#__RefHeading___Toc442057_2026549522">WDFAC</text:a></text:span> Keyword Description</text:p>
+    <text:p text:style-name="P58">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">12.3.259.1</text:sequence>: WDFAC Keyword Description</text:p>
```

Comment:
DO we need to WDFAC guard  to prevent scripts from adding links again. @hakonhagland ?
```suggestion
    <text:p text:style-name="P58">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">12.3.259.1</text:sequence>: <text:span text:style-name="NotKeyword">WDFAC</text:span> Keyword Description</text:p>
```